.travis.yml

```diff
@@ -16,5 +31,18 @@ script:
   - commitlint-travis
   - ng lint
   - ng build components
-  - ng test components --watch false --browsers ChromeHeadless
-  - ng build --prod --baseHref=/components/
+  - ng test components --watch=false --no-progress --browsers=ChromeHeadlessNoSandbox
+  - ng build --prod --baseHref=https://prosoft-edv.github.io/components/
+
+notifications:
+  email:
+    on_failure: change
+    on_success: change
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  local_dir: dist/prosoft-components-demo
+  on:
+    branch: master

```

Comment:
does this add a commit to the master branch with the updated dist folder, or how does this work?

Reply:
this means, that the gh-pages branch is only updated, if a commit in master branch is made.
see https://docs.travis-ci.com/user/deployment/pages/

Reply:
ah, ok :)